database/database.go

```diff
@@ -0,0 +1,104 @@
+package database
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+
+	"log"
+)
+
+func Start() error {
+	err := exec.Command(os.Getenv("GOPATH")+"/bin/tiedot", "-mode=httpd", "-dir="+filepath.Join(config.Persistant, config.Db), "-port="+config.DbPort).Start()
+	if err != nil {
+		log.Println("Failed to start database")
+		return err
+	}
+
+	time.Sleep(1 * time.Second)
```

Comment:
Ouch that's a nasty hard-coded delay, what's the justification? Why do we delay? If this is _completely_ necessary and we have to do it, there should be a comment explaining why. The code in general is lacking comments.